plugins/common.go

```diff
@@ -0,0 +1,12 @@
+package plugins
+
+import (
+	"github.com/gluster/glusterd2/servers/rest/route"
+	"github.com/gluster/glusterd2/servers/sunrpc/program"
+)
+
+type Gd2Plugin interface{
```

Comment:
Ok, I will copy that instead of move. and change name as something else for example purpose.